docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM socrata/nodejs
+
+ENV GEO_IMPORT_ENV=prod
+ENV BABEL_DISABLE_CACHE=1
+EXPOSE 4444
+# Need git and gdal for node-srs
+RUN apt-get update
```

Comment:
The update and install need to be part of a single run command, otherwise you risk the install running without the update in certain situations.  eg. https://github.com/socrata/shipyard/blob/master/base/Dockerfile#L12